internal/utils.go

```diff
@@ -16,3 +23,29 @@ func min(x, y int) int {
 	}
 	return y
 }
+
+func GetSemVer(version string) (float64, error) {
+	if len(version) > 0 {
```

Comment:
You can simplify this code with:
```
	res := semVerRegex.FindStringSubmatch(version)
	v := fmt.Sprintf("%s.%02s%03s\n", res[1], res[2], res[3])
```

play: https://play.golang.org/p/s-WXMWlfr1I

Reply:
Thanks, this is a lot simpler, made the change.